### DISCUSSION TITLE
add source arn or account as metadata

### DISCUSSION THREAD

**Discussion by dietmar-mueller** [iann0036/former2#381] [Ideas]
i use the fromer2 cli to reverse engineer existing aws services. for documentation purposes, i would need specific information on the origin of the data.

as output format i use the cloudfront format and i already see the prop metadata with the hint for generation.

is there a way to also output the arn, organisation or account prop of the corresponding service with the metadata?

**Comment by iann0036** [comment]
Hi @dietmar-mueller,

If you use the `--output-raw-data` option, you'll get as much information as is available. That'll sometimes have information about the above, but not always.
